src/STW.SubmissionApi/STW.SubmissionApi.sln

```diff
@@ -0,0 +1,24 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
The `.sln` should live in the `src` folder above the project folders. You'll also have to change some of the paths below once that's done.